src/sql/src/statements/copy.rs

```diff
@@ -44,6 +52,52 @@ impl CopyTable {
         &self.format
     }
 }
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct CopyTableFrom {
+    table_name: ObjectName,
+    format: Format,
+    connection: Option<HashMap<String, String>>,
```

Comment:
Why not just use HashMap?
```suggestion
    connection: HashMap<String, String>,
```